embed-code-go/configuration/configuration.go

```diff
@@ -16,7 +16,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// The configuration of the plugin.
+// Package configuration contains configuration of the plugin.
```

Comment:
@olena-zmiiova I beg your pardon, but why remove the articles which were used properly?

Reply:
@armiol, the `Go` package documentation requires the name of a package at the beginning, so I added it.